### DISCUSSION TITLE
Record to file while streaming

### DISCUSSION THREAD

**Discussion by mike-mathieu** [shogo4405/HaishinKit.dart#3] [Q&A] [answered]
Thanks for putting together this library (the only flutter RTMP library whose example app actually worked!). I had a quick question: is it possible to record to a local file while streaming with this library? If not is this something that may be added in the future?

**Comment by shogo4405** [comment, marked as answer]
No way. Currently not supported yet. I have a plan.
- iOS has feature. https://github.com/shogo4405/HaishinKit.swift/pull/1035
- Android has no feature. 

Thank you.

**Reply by imajared** [reply]
Hi @shogo4405  is it possible to pay you to implement this feature?

**Comment by user97116** [comment]
Camera recording not found
